src/node/desktop/CMakeLists.txt

```diff
@@ -191,6 +191,14 @@ if(WIN32)
       DESTINATION "${RSTUDIO_INSTALL_ELECTRON}"
    )
 
+   set(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP TRUE)
```

Comment:
Can we add a comment why we're doing this here?

Reply:
I also wonder if we need this for https://github.com/rstudio/rstudio/blob/64ee43493bae9754916b9f18f91fe79a6ba9b8a9/src/cpp/desktop/CMakeLists.txt#L519-L521.

Reply:
Ah, the CMake documentation at https://cmake.org/cmake/help/latest/module/InstallRequiredSystemLibraries.html makes it more clear:

> CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP
> Set to TRUE to skip calling the [install(PROGRAMS)](https://cmake.org/cmake/help/latest/command/install.html#command:install) command to allow the includer to specify its own install rule, using the value of CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS to get the list of libraries.

In other words, we set that if we want to provide our own `install()` command.